pkg/utils/util.go

```diff
@@ -78,6 +78,22 @@ const (
 	// backing the snapshot content.
 	AnnVolumeSnapshotBeingDeleted = "snapshot.storage.kubernetes.io/volumesnapshot-being-deleted"
 
+	// AnnVolumeSnapshotBeingCreated annotation applies to VolumeSnapshotContents.
+	// If it is set and value is "yes", it indicates that the csi-snapshotter
+	// sidecar has sent the create snapshot request to the storage system and
+	// is waiting for a response of success or failure.
+	// This annotation will be set to "no" if the driver's CreateSnapshot
+	// CSI function returns success or failure. If the create snapshot
+	// request times out, retry will happen and the annotation value will
+	// stay as "yes".
+	// This only applies to dynamic provisioning of snapshots because
+	// the create snapshot CSI method will not be called for pre-provisioned
+	// snapshots.
+	AnnVolumeSnapshotBeingCreated = "snapshot.storage.kubernetes.io/volumesnapshot-being-created"
```

Comment:
lets follow the pattern for "AnnVolumeSnapshotBeingDeleted"? I am not a big fan of using YES_NO in annotation values :).

Reply:
That means we need to remove the annotation after driver has responded.  I have not seen the PV/PVC controller remove any annotations so not sure if that is a pattern, but I can certainly do that here.

Reply:
Done